docs/userguide/pyproject_config.rst

```diff
@@ -70,6 +70,14 @@ The ``project`` table contains metadata fields as described by the
    # ... other project metadata fields as listed in:
    #     https://packaging.python.org/en/latest/guides/writing-pyproject-toml/
 
+.. important::
+   Support for
+   :external+PyPUG:ref:`project.license-files <pyproject-toml-license-files>`
+   and SPDX license expressions in
+   :doc:`project.license <PyPUG:specifications/pyproject-toml>` (:pep:`639`)
```

Comment:
And here maybe this one?
https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license
```suggestion
   and SPDX license expressions in
   :external+PyPUG:ref:`project.license <license>` (:pep:`639`)
```

Reply:
Hope I used the correct link target. This page is quite useful for it
https://webknjaz.github.io/intersphinx-untangled/packaging.python.org/

Reply:
Thank you!